app/requirements/requirements.txt

```diff
@@ -136,7 +142,7 @@ django-o365mail==1.1.0
     # via -r requirements.in
 django-silk==5.4.0
     # via -r requirements.in
-django-storages==1.14.6
+django-storages[s3]==1.14.6
```

Comment:
Als ik de code binnenhaal en het draai zonder de AWS env variabelen, krijg ik de volgende error: 
`urllib3.exceptions.NewConnectionError: <botocore.awsrequest.AWSHTTPConnection object at 0x7fa387acb4a0>: Failed to establish a new connection: [Errno 111] Connection refused`

Mogelijk komt het door deze manier van installeren?
Het lijkt erop dat Django verwacht dat de default storage nu een S3 storage is, terwijl dit de filesystem storage zou moeten zijn.